const.go

```diff
@@ -101,25 +101,27 @@ var ConfToEnv = map[string]string{
 
 	// [Clients]
 
-	// [Clients.Command]
-	"clients.command.port": "CLIENTS_COMMAND_PORT",
+	// FIXME: these are wrong, should be core-command, core-data, and core-metadata, support-*
```

Comment:
ok - yes, as per https://github.com/edgexfoundry/app-functions-sdk-go/blob/master/app-service-template/res/configuration.toml#L73-L92 - thank you for updating that.